projects/frontend/data-pipelines/gui/projects/ui/src/app/app.component.html

```diff
@@ -3,8 +3,10 @@
    ~ SPDX-License-Identifier: Apache-2.0
   -->
 
-<clr-main-container [attr.data-automation]="!!idToken ? 'vdk' : null">
-    <ng-container *ngIf="idToken">
+<clr-main-container
+    [attr.data-automation]="!!skipAuth && !!idToken ? 'vdk' : null"
```

Comment:
It is correct (mind it's skipAuth rather than enable, due it's a good design practice to rather have a negating toggle). Also, I have again tested both use cases (auth enabled, and not) locally, and I am able to successfully auth and see the panel expected. For example
<img width="1791" alt="Screenshot 2023-04-26 at 18 48 16" src="https://user-images.githubusercontent.com/11415233/234631202-9922eda2-a77d-4937-b408-d215b15f1ef0.png">